contents/handbook/growth/sales/lead-scoring.md

```diff
@@ -0,0 +1,42 @@
+|                | 1000+                                                                                                              | 20    |
+| Ability to pay | Estimated Revenue $0m-$1m                                                                                          | 0     |
+|                | Estimated Revenue $1m-$10m                                                                                         | 5     |
+|                | Estimated Revenue $10m-$100m                                                                                       | 10    |
+|                | Estimated Revenue $100m+                                                                                           | 20    |
+| Role           | engineering                                                                                                        | 10    |
+|                | product                                                                                                            | 10    |
+|                | leadership/founder                                                                                                 | 10    |
+|                | marketing                                                                                                          | 5     |
+|                | other                                                                                                              | 0     |
+| Sub-role       | data_science_engineer                                                                                              | 10    |
+|                | project_engineer                                                                                                   | 10    |
+|                | software_engineer                                                                                                  | 10    |
+|                | web_engineer                                                                                                       | 10    |
+|                | founder/ceo                                                                                                        | 10    |                 
+|                | other                                                                                                              | 0     |
+| Country        | Austria, Canada, France, Germany, Japan, Norway, Sweden, USA                                                       | 10    |
+|                | Australia, Belgium, Estonia, Finland, Georgia, Guernsey, Netherlands, New Zealand, Poland, Portugal, Singapore, UK | 5     |
+|                | Other                                                                                                              | 0     |
+
+
+See this internal GitHub [issue](https://github.com/PostHog/company-internal/issues/1459) for discussion on how we arrived at the above scoring
```

Comment:
I'd either remove this sentence, or add anything useful into the Handbook itself (probably just remove, the issue doesn't seem to add that much?)